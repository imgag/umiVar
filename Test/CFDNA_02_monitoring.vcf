##fileformat=VCFv4.1
##fileDate=07/07/2021
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
chr7	55259515	M	T	G	.	PASS	AF=0.008567
chr7	124392512	M	C	T	.	PASS	AF=0.789176
chr7	137029838	ID	T	C	.	PASS	AF=0.724108
chr7	140453136	ID	A	T	.	PASS	AF=0.463912