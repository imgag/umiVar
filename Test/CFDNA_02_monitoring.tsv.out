CHROM	POS	REF	ALT	DP_HQ	REF_COUNT	ALT_COUNT	AF	Multi_REF	Multi_ALT	Multi_AF	Pval	FDR	STRAND	FISHER	OAAC	MAL	Sequence_Context	Homopolymer	FILTER
chr7	124392512	C	T	8233	4601	3631	0.44103	3277	2608	0.443161	0.00e+00	0.00e+00	1934-1697-2431-2170	1.00e+00	1	1.00e+00	CTTTACATATG	0	PASS
chr7	55259515	T	G	8480	8404	72	0.008491	5798	45	0.007702	6.77e-05	1.09e-01	42-30-4350-4054	1.00e+00	4	1.00e+00	TGGGCTGGCCA	0	FDR
